cpp/simulations/2021_vaccination_sarscov2_delta_germany.cpp

```diff
@@ -717,14 +593,14 @@ mio::IOResult<void> run(RunMode mode, const fs::path& data_dir, const fs::path&
         temp_date = mio::Date(2021, 6, 6);
     }
     const auto start_date   = temp_date;
-    const auto num_days_sim = 90.0;
+    const auto num_days_sim = 20.0;
```

Comment:
```suggestion
    const auto num_days_sim = 90.0;
```

I think you changed the values for testing :)